src/scss/components/_cards.scss

```diff
@@ -17,8 +17,8 @@ $card-padding                                   : spacing(x2) !default;
 $card-radius                                    : 4px !default;
 $card-radius--large                             : 8px !default;
 $card-borderColor                               : $grey--lightest !default;
-$card-section-margin                            : spacing(x4) * 2 !default;
-$card-section-margin--aboveMid                  : spacing(x4) !default;
+$card-section-margin                            : spacing(x4) !default;
+$card-section-margin--aboveMid                  : spacing(x4) * 2 !default;
```

Comment:
Would it be better to add a `x8` option to the spacing map?

Reply:
Will do, I'll as a `x7` as well, doesn't feel right to skip the lucky number! 7️⃣